pmm-app/tests/verifyRemoteInstances_test.js

```diff
@@ -144,5 +142,72 @@ Scenario(
     pmmInventoryPage.deleteWithForceOpt();
     pmmInventoryPage.existsByid(agentIDToDelete, true);
     await pmmInventoryPage.checkAllNotDeletedAgents(agentID);
-  }
+  },
+);
+
+Scenario(
+  'PMM-T371 - Verify sorting in Inventory page(Services tab) @not-pr-pipeline',
```

Comment:
I think we can move these set of Scenarios as part of our nightly job, we need to make sure only the most critical ones are part of regular FB, so could you please add `@not-ui-pipeline @nightly` to all three